### PR TITLE
[devtools-translate] Russian - What's New in DevTools (Chrome 110)

### LINKED ISSUE #5280: [devtools-translate] Russian - What's New in DevTools (Chrome 110)
Post: https://developer.chrome.com/blog/new-in-devtools-110/
Translated by: Feb-23

Translator: @solarrust
Reviewers: @solarrust, @webmaxru, @kateryna-prokopenko

### PR DESCRIPTION
**Pull request by solarrust**
Fixes #5280

### CODE REVIEW

File: site/ru/blog/new-in-devtools-110/index.md [hunk 1]
```diff
@@ -111,48 +128,64 @@ Chromium issues: [1383453](https://crbug.com/1383453), [1382752](https://crbug.c
...
 
 
 <!-- ## Miscellaneous highlights {: #misc } -->
+## Другие важные моменты {: #misc }
 
 <!-- These are some noteworthy fixes in this release: -->
+Вот ещё несколько заслуживающих внимания исправлений в этом выпуске:
 
 <!-- - A regression issue in DevTools, where it failed to stop at the `debugger` statement in inline scripts, has been resolved. ([1385374](https://crbug.com/1385374)) -->
+- Устранена проблема регрессии в DevTools, когда он не останавливался на операторе `debugger` во встроенных скриптах. ([1385374](https://crbug.com/1385374))
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-110/index.md:183]:
```suggestion
- Устранена регрессивная проблема, когда DevTools не останавливался на операторе `debugger` во встроенных скриптах. ([1385374](https://crbug.com/1385374))
```

File: site/ru/blog/new-in-devtools-110/index.md [hunk 2]
```diff
@@ -111,48 +128,64 @@ Chromium issues: [1383453](https://crbug.com/1383453), [1382752](https://crbug.c
...
 <!-- - A new **Console** setting that allows you to expand or collapse `console.trace()` messages by default. Toggle the settings via **Settings** > **Preferences** >  **Expand console.trace() messages by default**. ([1139616](https://crbug.com/1139616)) -->
+- Новая настройка в **Консоли** (Console), которая позволяет по умолчанию скрывать или разворачивать сообщения `console.trace()`. Опция доступна в **Настройки** (Settings) > **Параметры** (Preferences) > **Разворачивать сообщения `console.trace()` по умолчанию** (Expand console.trace() messages by default). ([1139616](https://crbug.com/1139616))
 <!-- - The [Snippets](/docs/devtools/javascript/snippets/) pane in the **Sources** panel supports enhanced autocomplete, similar to the **Console**. ([772949](https://crbug.com/772949))  -->
-  {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/thkb1CYO0yYiGHll7Yp8.png", alt="Autocomplete in Snippets.", width="800", height="417" %}
+- Панель [Фрагменты (Snippets)](/docs/devtools/javascript/snippets/) во вкладке **Источники** (Sources) теперь поддерживает автодополнение, по примеру **Консоли** (Console). ([772949](https://crbug.com/772949))
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-110/index.md:187]:
```suggestion
- Панель [Фрагменты (Snippets)](/docs/devtools/javascript/snippets/) во вкладке **Источники** (Sources) теперь поддерживает улучшенное автодополнение, по примеру **Консоли** (Console). ([772949](https://crbug.com/772949))
```

**Review by kateryna-prokopenko — COMMENTED** [review]
Thanks @solarrust, left some comments!

**Comment by jecfish** [comment]
Thanks Kateryna, over to @webmaxru and @solarrust

**Review by AaronForinton — APPROVED** [review]

**Pull request merged**